R/RLearner_regr_randomForest.R

```diff
@@ -114,18 +129,39 @@ jackknifeStandardError = function(.learner, .model, .newdata, ...) {
   n = nrow(model$inbag)
   ntree = model$ntree
   pred = predict(model, newdata = .newdata, predict.all = TRUE)
-  oob = t(sapply(seq_len(n), function(i) model$inbag[i, ] == 0))
+  oob = model$inbag == 0
   jack_n = apply(oob, 1, function(x) rowMeans(pred$individual[, x, drop = FALSE]))
   if (is.vector(jack_n)) {
     jack_n = t(as.matrix(jack_n))
   }
   jack = (n - 1) / n * rowSums((jack_n - pred$aggregate)^2)
   bias = (exp(1) - 1) * n / ntree^2 * rowSums((pred$individual - pred$aggregate)^2)
-  jab = jack - bias
-  jab[jab < 0] = 0
+  jab = pmax(jack - bias, 0)
```

Comment:
this is just a more efficient replacement no? you should get a logical matrix in both cases?